English_transcripts/school/Epic Quest/sharedAssets/P_C_M_ENG_L3_AU_numline_dota_20_b.txt

```diff
@@ -0,0 +1 @@
+. A is right after number 5. 
```

Comment:
Remove the full stop at the beginning.

Reply:
These full stops are intentional as per the English audio.